forge-gui/res/cardsfolder/upcoming/lictor.txt

```diff
@@ -0,0 +1,10 @@
+Name:Lictor
+ManaCost:3 G
+Types:Creature Tyranid
+PT:3/3
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | CheckSVar$ X | SVarCompare$ GE1 | TriggerDescription$ Pheromone Trail — When CARDNAME enters the battlefield, if a creature entered the battlefield under an opponent's control this turn, create a 3/3 green Tyranid Warrior creature token with trample.
+SVar:TrigToken:DB$ Token | TokenScript$ g_3_3_tyranid_warrior | CheckSVar$ X | SVarCompare$ GE1
```

Comment:
remove svar check, you already did it in line 6 and these params wouldn't do work here anyway